### ISSUE TITLE
GM_download does not download to specified folder.

### ISSUE THREAD

**Issue by frozenbits** [Tampermonkey/tampermonkey#1010]
Note:
It _was_ working up to a few days ago. I thought firefox or tampermonkey had an update but no, that doesn't seem like the case. I did not see anything in the firefox changelog or tampermonkey changelog that says anything about this.
I also compared it with chrome, and it's still working there. So far, I have to guess this case is isolated to Firefox.
Also, both are currently using the "Browser API" download mode.

## Expected Behavior
 When GM_download is called with the "name" parameter set with a forward slash "/", the downloaded file should be created inside that folder.
Example: 
```name: "sample/sample_image.jpg"```
should result in the file created with the path:
```Downloads\sample\sample_image.jpg```

## Actual Behavior
The forward slash is instead renamed as a dash ("-").
Example:
```name: "sample/sample_image.jpg"```
the file is instead created as
```Downloads\sample-sample_image.jpg```

## Specifications

- Firefox: 79.0 (64-bit)
- Chromium: 84.0.4147.105 (Official Build) (64-bit) (chrome does not have any problem, just for comparison)
- TM: 4.116117
- OS: Windows 10 Build 18363.959
## Script

```js
// ==UserScript==
// @name         New Userscript
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        https://i.redd.it/xf13lv5n2qf51.jpg
// @grant        GM_download
// ==/UserScript==

    'use strict';
    let downloadArgs = {
        url: document.querySelector("img").currentSrc,
        name: "sample/sample_image.jpg"
    }
    GM_download(downloadArgs)

```

**Comment by Messere** [comment]
> > Same issue affects Chromium Edge browser (v. 84.0.522.63)
> 
> Then it might be a browser issue. Tampermonkey 4.10.6111 should work fine at MS Edge. Need to test...

False alarm. I've re-tested it today on other computer and everything worked fine. I didn't notice earlier that "download mode" was set to "default". Subdirectories work fine after setting it to "browser API" and granting appropriate permission.

**Comment by derjanb** [comment]
Fixed at 4.11.6119

**Comment by afgit** [comment]
So I took a deeper look into the settings: they were all ok.

But I was thinking naively that Brave and Chrome behave exactly the same - which is unfortunately not the case:
Trying again in Chrome, it works, but not in Brave 😒

So it's probably nothing Tampermonkey can do about, is it?
